web-common/src/features/dashboards/state-managers/selectors/measures.ts

```diff
@@ -51,11 +52,27 @@ export const isMeasureValidPercentOfTotal = ({
   };
 };
 
-export const filterBasicMeasures = (
-  measures: MetricsViewSpecMeasureV2[] | undefined,
-) => measures?.filter((m) => !m.window) ?? [];
+export const filteredBasicMeasures = ({
+  metricsSpecQueryResult,
+  dashboard,
+}: DashboardDataSources) => {
+  return (includeComparisonMeasure: boolean) => {
```

Comment:
1. It's quite hard to understand this function where it's invoked like `filteredBasicMeasures(dashData)(false)`. To get rid of the second function call, maybe we can split it up into two functions – one for comparisons, one without?
2. Rather than introducing a new term "basic" measure, can we use existing terms, like "simple" measure or "windowless" measure?

Reply:
Sounds good. Will just remove the boolean for now since it is always false.